### DISCUSSION TITLE
Version 1.15.1

### DISCUSSION THREAD

**Discussion by shazzaam7** [xenia-manager/xenia-manager#170] [General]
## What's Changed
* Fixed a bug where changing boxart wasn't working properly (PR https://github.com/xenia-manager/xenia-manager/pull/165) 
* When installing locally available patches, the original file won't be deleted anymore (It will just copy and paste the patches file to the correct folder) (PR https://github.com/xenia-manager/xenia-manager/pull/165)

**Full Changelog**: https://github.com/xenia-manager/xenia-manager/compare/1.15.0...1.15.1

<hr /><em>This discussion was created from the release <a href='https://github.com/xenia-manager/xenia-manager/releases/tag/1.15.1'>Version 1.15.1</a>.</em>

**Comment by RobertJG60** [comment]
this update broke dlc.

even after reinstalling the dlc, it doesn't show up in soltrio solitare

**Reply by RobertJG60** [reply]
how do I go back to 1.15?

**Reply by shazzaam7** [reply]
I haven't touched anything in the content related code in this update, and since the profile update on Xenia Canary this has been broken as stated in the issues tab. 
You can download the older release and manually replace the files inside of your Xenia Manager folder.

**Reply by RobertJG60** [reply]
it's also making you create a profile or you can't play

**Reply by RobertJG60** [reply]
Thank you, switching the stable fixed it.

**Comment by NuclearGeneral** [comment]
Box art is still not working in the latest 1.15.1 build.

**Comment by SirBigzalot** [comment]
This isnt a problem or anything, but i just wanted to express how genuinely grateful I am for you and this creation. Xenia Manager is so damn awesome. Thank you so much for this creation man its genuinely so amazing.

**Reply by SRC267** [reply]
What's the best thing you like about it?

**Reply by SirBigzalot** [reply]
I love how its so well set up, but also so simplistic and straight forward. Using the marketplace library to identify games is something i find so cool. The covers on the front page make me want to install a ton of roms to have my own digital collection. My favorite thing has to be how easy it is to change the graphics settings. It makes the quality of my experience running 360 games so much better and easier. Its so much better than opening files and changing them yourself. Tweaking RDR and making it look so nice was such a fun thing to play around with. Overall, it is genuinely so much better than normal xenia, in EVERY way!

**Reply by SRC267** [reply]
Yea, plus the easy way of applying game patches, title updates and dlc. Take away the headache of setting up each game.
